javascript-this/student.js

```diff
@@ -4,10 +4,11 @@ var student = {
   lastName: 'macrae',
   subject: 'JS',
   getFullName: function () {
+    // console.log(this.firstName);
     return this.firstName + ' ' + this.lastName;
   },
   getIntroduction: function () {
+    // console.log(this.getFullName());
     return 'Hello, my name is' + ' ' + this.firstName + ' ' + this.lastName + ' ' + 'and I am studying' + ' ' + this.subject + '.';
   }
-
 };
```

Comment:
Make sure you aren't committing files from old exercises! You should only be adding files for this current exercise. If you plan on going back to play with previous exercises, please go to that branch.